### ISSUE TITLE
STM or GD chip?

### ISSUE THREAD

**Issue by az1984** [Bully85/Sovol-SV04-Klipper#9]
In the guides, you state:

"Before starting, verify whether your chip is STM or GD. You can do this by either opening your electric box and checking manually, or by navigating to 'Machine -> System Loads -> mcu' in Mainsail."

But I don't understand what to do with that information. The README.md doesn't mention doing X if it's STM and Y if it's GTD. I also didn't see two distinct firmware files... Is the project only compatible with one or the other?

Can you please clarify?

**Comment by Bully85** [comment]
You're right. It's a bit unmanageable. I'll review this over the weekend. The firmware is the same in both cases. Only in the Printer.cfg must the correct one be activated under z axis and the wrong one commented out or deleted.
